repl/src/test/scala/com/cloudera/livy/repl/Python3InterpreterSpec.scala

```diff
@@ -208,15 +208,5 @@ class Python3InterpreterSpec extends BaseInterpreterSpec {
     ))
   }
 
-  // Scalastyle is treating unicode escape as non ascii characters. Turn off the check.
-  // scalastyle:off non.ascii.character.disallowed
-  it should "print unicode correctly" in withInterpreter { intp =>
-    intp.execute("print(u\"\u263A\")") should equal(Interpreter.ExecuteSuccess(
```

Comment:
that time, i'm not sure that case need to be python3.
but. you right. i will fixed code next week.